cmd/git-sync/main.go

```diff
@@ -183,6 +189,36 @@ func envDuration(key string, def time.Duration) time.Duration {
 	return def
 }
 
+func sanitizeSpecificTime(t string) (h int, m int) {
+	if uniseg.GraphemeClusterCount(t) != 4 {
+		err := errors.New("error=")
+		log.Error(err, "--specific-time value not in military format, e.g. 1500 or 0130")
+		os.Exit(1)
+	}
+
+	tsplit := strings.Split(t, "")
+	h, _ = strconv.Atoi(strings.Join(tsplit[0:2], ""))
+	m, _ = strconv.Atoi(strings.Join(tsplit[2:], ""))
+
+	return h, m
```

Comment:
this method seems kind of fragile. It might be better to utilize the `time` package a bit more.

https://gobyexample.com/time-formatting-parsing
https://stackoverflow.com/questions/37581088/how-to-validate-hhmma-time-format-in-go

so instead of `uniseg.GraphemeClusterCount(t) != 4` you can do `time.parse` and check if parsing gives you an error. you can specificy a format before you parse and you should be good to go.

you can then pass the `time` object back and later use it when comparing if its time to sync